kvdb-web/Cargo.toml

```diff
@@ -9,16 +9,16 @@ license = "GPL-3.0"
 edition = "2018"
 
 [dependencies]
-wasm-bindgen = "0.2.51"
-js-sys = "0.3.28"
+wasm-bindgen = "0.2.54"
+js-sys = "0.3.31"
 kvdb = { version = "0.1", path = "../kvdb" }
 kvdb-memorydb = { version = "0.1", path = "../kvdb-memorydb" }
 futures-preview = "0.3.0-alpha.19"
 log = "0.4.8"
 send_wrapper = "0.3.0"
 
 [dependencies.web-sys]
-version = "0.3.28"
+version = "0.3.31"
```

Comment:
`futures` crate version 0.3/0.3.1 released +- two weeks ago. Can be replaced with:

```diff
diff --git a/kvdb-web/Cargo.toml b/kvdb-web/Cargo.toml
index be35681..c9157d9 100644
--- a/kvdb-web/Cargo.toml
+++ b/kvdb-web/Cargo.toml
@@ -13,7 +13,7 @@ wasm-bindgen = "0.2.54"
 js-sys = "0.3.31"
 kvdb = { version = "0.1", path = "../kvdb" }
 kvdb-memorydb = { version = "0.1", path = "../kvdb-memorydb" }
-futures-preview = "0.3.0-alpha.19"
+futures = "0.3"
 log = "0.4.8"
 send_wrapper = "0.3.0"
```

Reply:
Thanks!